tests/main/xdg-open-on-core/task.yaml

```diff
@@ -0,0 +1,26 @@
+summary: Ensure snap userd returns an error on Ubuntu Core
+
+systems:
+    - ubuntu-core-*
```

Comment:
if this test is meant to be run on only core devices, you could move it to the `tests/core/...` suite instead where only ubuntu core systems are available to run